ServerCore/Pages/Responses/Delete.cshtml.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Microsoft.EntityFrameworkCore;
+using ServerCore.DataModel;
+using ServerCore.Models;
+
+namespace ServerCore.Pages.Responses
+{
+    public class DeleteModel : PageModel
+    {
+        private readonly ServerCore.Models.PuzzleServerContext _context;
+
+        public DeleteModel(ServerCore.Models.PuzzleServerContext context)
+        {
+            _context = context;
+        }
+
+        [BindProperty]
+        public Response PuzzleResponse { get; set; }
+
+        public async Task<IActionResult> OnGetAsync(int? id)
```

Comment:
We're modifying these pages to have the parameters not be optional, unless they really really are. The NotFound case on line 27 will then be automatically handled.